tests/integration/src/test/scala/mesosphere/marathon/integration/UpgradeIntegrationTest.scala

```diff
@@ -32,72 +31,57 @@ class UpgradeIntegrationTest extends AkkaIntegrationTest with MesosClusterTest w
 
   val zkURLBase = s"zk://${zkServer.connectUri}/marathon-$suiteName"
 
-  val marathon149Artifact = MarathonArtifact("1.4.9", "marathon-1.4.9.tgz")
-  val marathon1515Artifact = MarathonArtifact("1.5.15", "marathon-1.5.15.tgz")
-  val marathon16549Artifact = MarathonArtifact("1.6.549", "marathon-1.6.549-aabf74302.tgz")
+  val marathonMinus3Artifact = MarathonArtifact(SemVer(1, 6, 567, Some("2d8b3e438")))
```

Comment:
This one is tricky. We used 1.4.9 and 1.5.15 to guarantee that we can still consume the task ids created by these versions. That was the original idea of the test. Can we produce Mesos tasks started by Marathon 1.4.9 and 1.5.15 without using these?

Reply:
I think I'll feel okay with unit tests to assert this going forward

Reply:
I'm not aware of any unit test that verifies task id parsing for all possible ids and labels.